tests/integration/test_search_views.py

```diff
@@ -59,3 +61,11 @@ def test_search_jobs_is_there(app):
 def test_search_falls_back_to_hep(app):
     with app.test_client() as client:
         assert client.get('/search').status_code == 200
+
+
+@patch('inspirehep.modules.search.query.current_app')
+def test_search_logs(current_app_mock, app):
+    log_output = '{\n    "query": {\n        "bool": {\n            "filter": [\n                {\n                    "bool": {\n                        "must_not": [\n                            {\n                                "match": {\n                                    "_collections": "Another Restricted Collection"\n                                }\n                            }, \n                            {\n                                "match": {\n                                    "_collections": "Restricted Collection"\n                                }\n                            }\n                        ], \n                        "must": [\n                            {\n                                "match": {\n                                    "_collections": "Literature"\n                                }\n                            }\n                        ]\n                    }\n                }\n            ], \n            "must": [\n                {\n                    "multi_match": {\n                        "query": "", \n                        "fields": [\n                            "title^3", \n                            "title.raw^10", \n                            "abstract^2", \n                            "abstract.raw^4", \n                            "author^10", \n                            "author.raw^15", \n                            "reportnumber^10", \n                            "eprint^10", \n                            "doi^10"\n                        ], \n                        "zero_terms_query": "all"\n                    }\n                }\n            ]\n        }\n    }, \n    "from": 0, \n    "size": 10\n}'
```

Comment:
In any case I need to check a bit more because I added to the log output `Another Restricted Collection` to make Travis happy but I think that is a leftover from a previous test and should not be there.